src/AppContext.tsx

```diff
@@ -20,6 +21,7 @@ enum storeKey {
 }
 type AppState = {
   isLoading: boolean;
+  extendedOnboardingOnboarded: boolean;
```

Comment:
Is not a bit intuitive the meaning of this attr.

Reply:
Yeah I agree. Problem was that the pattern in `AppContext` is a flag with `Onboarded` at the end, and also there is a flag in remote config `enable_extended_onboarding`. So either break the pattern, introduce a new (and kinda duplicate) remote config variable, or accept `extendedOnboardingOnboarded` 😬